tests/pkg_semtech-loramac/Makefile

```diff
@@ -0,0 +1,22 @@
+APPLICATION = pkg_semtech-loramac
+include ../Makefile.tests_common
+
+BOARD ?= b-l072z-lrwan1
```

Comment:
This must go above including Makefile.tests_common, otherwise that file sets "native"